new-build-specification.rst

```diff
@@ -0,0 +1,99 @@
+.. post:: Oct 14, 2021
+   :tags: feature, builders
+   :author: Juan Luis
+   :location: MAD
+
+.. meta::
+   :description lang=en:
+      You can now use a build image based on Ubuntu 20.04 on Read the Docs
+      with some extra features.
+
+New build specification
+=======================
```

Comment:
This is not a descriptive title. Perhaps `New Ubuntu 20.04 build configurations`?